### ISSUE TITLE
Add support for multiple ORM

### ISSUE THREAD

**Issue by hason** [it-blaster/form-constructor-bundle#1]
The best practises in bundles is to provide model layer that is independent on a concrete implementation of ORM. Can you please make Propel as a suggested dependency? For inspiration see https://github.com/FriendsOfSymfony/FOSUserBundle

**Comment by fenrizbes** [comment]
Yeah, I'm planning it. Currently I'm implementing the front-end validation for constructed forms. After that I'm going to do some refactoring, and a model layer is included in it.
